.gitignore

```diff
@@ -8,3 +8,5 @@
 /.nyc_output/
 /unexpected.js
 /unexpected.js.map
+/unexpected-deno.js
```

Comment:
If we do end up generating this, I wonder if we should just call the file `esm` and the. allow users to `import expect from “unexpected/esm"`.

Also need to check what the options are for making these files accessible to the deno registry.

Reply:
`import expect from “unexpected/esm"` won't really help deno users, as they aren't using npm. It seems like we'll need to get unexpected added to https://github.com/denoland/registry/blob/master/src/database.json, check the generated file into git, and then hand out an url like https://deno.land/x/unexpected/unexpected-deno.js

Reply:
That’s a fair point - currently the deno registry is limited to pointing you to the project but the you must be able to import a file that is checked in. That said, the esm build is still generic so if we do check it in I think we should consider renaming :)

Reply:
Absolutely, not a lot of thought went into the name :)